hoot-js/src/main/cpp/hoot/js/JavaScriptTranslator.cpp

```diff
@@ -160,7 +160,7 @@ vector<JavaScriptTranslator::TranslatedFeature> JavaScriptTranslator::_createAll
   QVariantList list)
 {
   vector<TranslatedFeature> result;
-  result.reserve(list.size());
+  //result.reserve(list.size());
```

Comment:
Why comment this out? Either leave it or remove it. I don't think it does any harm, but doesn't help much either.